ehrapy/data/_datasets.py

```diff
@@ -168,13 +177,53 @@ def diabetes_130(
     return adata
 
 
+def diabetes_130_fairlearn(
+    encoded: bool = False,
+    columns_obs_only: dict[str, list[str]] | list[str] | None = None,
+) -> AnnData:
+    """Loads the preprocessed diabetes-130 dataset by fairlearn
+
+    This is a wrapper around the `fairlearn.datasets.fetch_diabetes_hospital` function.
+
+    More details: http://archive.ics.uci.edu/ml/datasets/Diabetes+130-US+hospitals+for+years+1999-2008
+
+    Preprocessing: https://fairlearn.org/v0.10/api_reference/generated/fairlearn.datasets.fetch_diabetes_hospital.html#fairlearn.datasets.fetch_diabetes_hospital
+
+    Args:
+        encoded: Whether to return an already encoded object
+        columns_obs_only: Columns to include in obs only and not X.
+
+    Returns:
+        :class:`~anndata.AnnData` object of the Diabetes 130 dataset processed by the fairlearn team
+
+    Examples:
+        >>> import ehrapy as ep
+        >>> adata = ep.dt.diabetes_130_fairlearn()
+    """
+
+    # df = fetch_diabetes_hospital(data_home=f"{ehrapy_settings.datasetdir}/diabetic_data_fairlearn.csv").data
+    if not Path.exists(Path(f"{ehrapy_settings.datasetdir}/diabetes_130_fairlearn.csv")):
+        df = fetch_diabetes_hospital().data
+        df.to_csv(f"{ehrapy_settings.datasetdir}/diabetes_130_fairlearn.csv", index=False)
+
+    adata = read_csv(
+        dataset_path=f"{ehrapy_settings.datasetdir}/diabetes_130_fairlearn.csv", columns_obs_only=columns_obs_only
+    )
+
+    if encoded:
+        return encode(adata, autodetect=True)
+
+    return adata
+
+
 def chronic_kidney_disease(
     encoded: bool = False,
     columns_obs_only: dict[str, list[str]] | list[str] | None = None,
 ) -> AnnData:  # pragma: no cover
     """Loads the Chronic Kidney Disease dataset
 
     More details: https://archive.ics.uci.edu/ml/datasets/Chronic_Kidney_Disease
+
```

Comment:
If no space here, the two lines are just separated by a space instead of a linebreak